packages/devkit/src/utils/config-utils.ts

```diff
@@ -54,15 +54,23 @@ export function getRootTsConfigFileName(): string | null {
   return null;
 }
 
+const packageInstallationDirectories = ['node_modules', '.yarn'];
+
+export function clearRequireCache() {
+  for (const k of Object.keys(require.cache)) {
+    if (!packageInstallationDirectories.some((dir) => k.includes(dir))) {
```

Comment:
Please use the `relative` + `startsWith` check rather than `includes`

Reply:
`relative` + `startsWith` wouldn't handle npm workspaces right? e.g. `apps/my-app/node_modules/jest` if the app has a different version of jest installed